CRM/Upgrade/Incremental/php/SixZero.php

```diff
@@ -37,6 +37,37 @@ public function upgrade_6_0_alpha1($rev): void {
       "timestamp NULL DEFAULT NULL COMMENT 'Date this membership modification action was logged.'",
       FALSE
     );
+    $this->addTask('Set a default activity priority', 'addActivityPriorityDefault');
+  }
+
+  /**
+   * This task sets the Normal option as the default activity status.
+   * It was previously hardcoded in Form and BAO files.
+   *
+   * @return bool
+   */
+  public static function addActivityPriorityDefault() {
+    // Check if there is a default option already set
+    // it might not be 'Normal'
```

Comment:
Checking for existing is smart. Note this whole function can be done in one query:

```sql
UPDATE civicrm_option_value ov
INNER JOIN civicrm_option_group og ON (og.id = ov.option_group_id AND og.name='priority')
SET ov.is_default = 1 WHERE ov.name='Normal'
-- but only if there is no default already set
AND NOT EXISTS
 (SELECT ov2.id FROM civicrm_option_value ov2
   INNER JOIN civicrm_option_group og2 ON (og2.id = ov2.option_group_id AND og2.name='priority')
   WHERE ov2.is_default=1
 );
```

Reply:
Added to the PR. Thanks :)